python/ray/ml/integrations/torch/torch_utils.py

```diff
@@ -0,0 +1,84 @@
+from typing import Optional, Union, List
+
+import pandas as pd
+import torch
+
+
+def convert_pandas_to_torch_tensor(
```

Comment:
This seems to be a data conversion, would it make sense to place in python/ray/data/ ?

Reply:
Hmm since `ray.data` is currently for Datasets-only code, and this is a utility for creating Torch tensors with no Datasets dependencies, I think this placement in Torch utilities makes sense.

Reply:
+1 to @clarkzinzow, we want to use this utility in Ray ML code, and we don't want ray.data to be a dependency.

Reply:
I think we might evaluate what the ray.data is designed to have - if it's Dataset-only, maybe we can make it more accurate as ray.dataset (not suggesting we should do it now).
For dependency on ray.data, any concern to depend on it? It looks the dependency already exists.

Reply:
@naijoaix hmm there shouldn't be a dependency on ray.data from ray.ml, where do you see this?

Reply:
I don't know how python/ray/ml is different from others like train/tune/serve (perhaps a README under those directories may help), but saw this one: https://sourcegraph.com/github.com/ray-project/ray/-/blob/python/ray/ml/preprocessor.py?L3
My understanding is ML code will use data (hence Dataset) since the algorithm works that way, no?

Reply:
Ah got it, yeah `ray.ml.preprocessor` does depend on `ray.data`, though the intended usage for `convert_pandas_to_torch_tensor` is for predictors, which I don't think we want to depend on `ray.data`.

But I agree with you though, the dependency story for ray.ml is not fully fleshed out yet. This is still in alpha release though, so we will make sure to smooth this out over time. cc @ericl